test/model/avatar_url_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:checks/checks.dart';
+import 'package:test/scaffolding.dart';
+import 'package:zulip/model/avatar_url.dart';
+
+import '../example_data.dart' as eg;
+
+void main() {
+  final user = eg.user();
+  final account = eg.account(user: user);
+  
+  double defaultSize = 30;
+  double largeSize = 120;
+
+  group('GravatarUrl', () {
+    test('gravatar url', () {
+      String url = '${GravatarUrl.origin}/avatar/1234';
```

Comment:
These numbers and this string are more examples of these points from https://github.com/zulip/zulip-flutter/pull/534#discussion_r1503273346 :
> * local variables are final or const if there's no intention to mutate them
> * local variables don't need type annotations unless there's a reason type inference wouldn't give the same answer

In general, when you get a piece of code-review feedback in one place, it's important to look for other places in your PR (or other open and future PRs) where the same comment applies. That's a key ingredient in making it easy for maintainers to review and merge your PRs.

Reply:
[8e9cf2e](https://github.com/zulip/zulip-flutter/pull/534/commits/8e9cf2ebfb70130ec8c00136d758e2dfdcb4677d)